bottles/backend/wine/winecommand.py

```diff
@@ -14,6 +14,7 @@
     obs_vkc_available, vkbasalt_available, vmtouch_available
 from bottles.backend.logger import Logger
 from bottles.frontend.utils.threading import RunAsync
+from bottles.backend.health import HealthChecker
```

Comment:
I suggest to use the GPUUtills instance defined here
https://github.com/bottlesdevs/Bottles/blob/046944ed86dc633d67589a96ffbb76e7661385e0/bottles/backend/wine/winecommand.py#L147

This provide the full GPU configuration: https://github.com/bottlesdevs/Bottles/blob/046944ed86dc633d67589a96ffbb76e7661385e0/bottles/backend/utils/gpu.py#L89

This allow checking for bot prime and classic setup, e.g. 
https://github.com/bottlesdevs/Bottles/blob/046944ed86dc633d67589a96ffbb76e7661385e0/bottles/backend/wine/winecommand.py#L324